api/envoy/config/route/v3/route_components.proto

```diff
@@ -1047,6 +1051,9 @@ message RouteAction {
   // it'll take precedence over the virtual host level hedge policy entirely
   // (e.g.: policies are not merged, most internal one becomes the enforced policy).
   HedgePolicy hedge_policy = 27;
+
+  // Specifies the maximum stream duration. If set, *timeout* is ignored.
```

Comment:
I thought we said that this was going to be independent of the `timeout` field, since it's basically a completely independent timeout starting from a different point?  Or am I remembering wrong?

Reply:
Hmm, yes I think you're right.  I was thinking we said that because gRPC couldn't support `timeout`, this should disable it.  But, we will need to document that we don't support `timeout` no matter what, in case this isn't set.